css-view-transitions-1/Overview.bs

```diff
@@ -1172,7 +1172,7 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 	<dl dfn-for="captured element">
 		: <dfn>old image</dfn>
-		:: an 2D bitmap or null. Initially null.
+		:: a 2D bitmap, "`live`" or null. Initially null.
```

Comment:
What would `old image` hold in the live case then? I actually think this is a good way to represent it and it's common in other standards (e.g. `fetch` callbacks pass `failure`, a network error, or a response).

Reply:
> What would old image hold in the live case then?

The old image can stay null, since we didn't capture anything. And a bool on captured element like, "should display new live image for old pseudo".

I find the pattern of strongly typed data types in the spec makes it more readable, could be my C++ brain. 😅 So would prefer it unless you feel strongly otherwise.

Reply:
No worries. Actually I need to do this anyway because once we update from the actual live snapshot that `live` enum is replaced with an image anyway.

Reply:
Hmm actually trying to refactor this makes it so that I have to check for this boolean in multiple places (where we check for `old image is null`). I think the variant approach is a lot cleaner...

Reply:
Maybe the bool can be "should generate old pseudo"? That's what old image != null implies which we want to change right. Then we'd only use old image when we need the pixels.

Reply:
Fine, done.